src/index.html

```diff
@@ -11,7 +11,138 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="POST">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label class="form_field">
+          Surname:
+          <input
+          type="text"
+          name="surname"
+          autocomplete="off"
+          >
+        </label>
+
+        <label class="form_field">
+          Name:
+          <input
+            name="name"
+            type="text"
+            autocomplete="off"
+            minlength = "2"
+            maxlength = "30"
+            required
+          >
+        </label>
+
+        <label class="form_field">
+          How old are You?
+          <input
+            name="age"
+            type="number"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+
+        <label class="form_field">
+          Full date of birth:
+          <input type="date" name="date_of_birth">
+        </label>
+
+        <label class="form_field">
+          I accept the term of the agreement:
+          <input type="checkbox" name="accept_terms">
+        </label>
+      </fieldset>
+    </form>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="POST">
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+        <label class="form_field">
+          E-mail:
+          <input type="email" placeholder="email@example.com" name="email">
```

Comment:
more than 2-3 props => move each to new line

```suggestion
          <input
            type="email"
            placeholder="email@example.com"
            name="email"
          >
```